src/codegen/llvm/codegen_llvm_visitor.cpp

```diff
@@ -65,6 +65,50 @@ static bool can_vectorize(const ast::CodegenForStatement& statement, symtab::Sym
     return unsupported.empty() && supported.size() <= 1;
 }
 
+#if LLVM_VERSION_MAJOR >= 13
+void CodegenLLVMVisitor::add_vectorizable_functions_from_vec_lib(llvm::TargetLibraryInfoImpl& tli,
+                                                                 llvm::Triple& triple) {
+    // Since LLVM does not support SLEEF as a vector library yet, process it separately.
+    if (vector_library == "SLEEF") {
+        // Populate function definitions for AArch64 architecture.
```

Comment:
below functions are `AArch64`? If so, we can also add functions for x86_64, right?

(reason being SVML is intel proprietary library and can not be `shipped` with package like NMODL. `sleef` we can distribute with NMODL if we decide to!)

Reply:
From what I have read on sleef, it has functions for both AArch64 and x86_64. Here I used the functions from AArch64 which were used in that non-committed LLVM patch. But I think we can easily extend this to support x86! Maybe I should add a comment saying this!

Reply:
addressed